parse.go

```diff
@@ -106,13 +106,14 @@ func Parse(fs *flag.FlagSet, args []string, options ...Option) error {
 
 // Context contains private fields used during parsing.
 type Context struct {
-	configFile         string
-	configFileFlagName string
-	configFileParser   ConfigFileParser
-	envVarPrefix       string
-	envVarNoPrefix     bool
-	envVarIgnoreCommas bool
-	ignoreUndefined    bool
+	configFile             string
+	configFileFlagName     string
+	configFileParser       ConfigFileParser
+	configFileAllowMissing bool
```

Comment:
Minor nit: I think this parses better as

```suggestion
	allowMissingConfigFile bool
```